pages.zh/common/locate.md

```diff
@@ -13,4 +13,4 @@
 
 - 重新建立文件数据索引数据库。如果要查找最近添加的文件，则需要执行此操作：
 
-`sudo /usr/libexec/locate.updatedb`
```

Comment:
Are you sure this isn't necessary on macOS? This seems pretty intentional, especially because it's the only difference to the Linux page.